src/mcp/tools/dataconnect/execute_dataconnect_graphql.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod";
+
+import { tool } from "../../tool.js";
+import * as client from "../../../dataconnect/dataplaneClient.js";
+import { pickService } from "../../../dataconnect/fileUtils.js";
+import { graphqlResponseToToolResponse } from "./converter.js";
+
+export const execute_dataconnect_graphql = tool(
```

Comment:
just `execute_graphql` [here](https://github.com/firebase/firebase-tools/blob/master/src/mcp/tools/index.ts#L30).

We adds the prefix, so it becomes `dataconnect_execute_graphql`.